.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "editor.codeActionsOnSave": null,
+  "editor.quickSuggestions": null
+}
```

Comment:
Did you mean to include this? Can you .gitignore it if not?

Reply:
git ignored